adapters/tappx/tappxtest/exemplary/single-banner-impression-extra.json

```diff
@@ -0,0 +1,124 @@
+{
+  "mockBidRequest": {
+    "id": "0000000000001",
+    "test": 1,
+    "imp": [
+      {
+        "id": "adunit-1",
+        "banner": {
+          "w": 300,
+          "h": 250
+        },
+        "ext": {
+         "bidder": {
+            "tappxkey": "pub-12345-android-9876",
+            "endpoint": "ZZ123456PS",
+            "host": "testing.ssp.tappx.com/rtb/v2/",
+            "mktag": "mktag-12345-android-9876"
```

Comment:
Can you please either update this test with `bcid` and `bcrid` or add a new test with them? I don't see these fields in any of the tests.

Reply:
Ok, we can add the params to the new test! 👍 
We add in the commit 09ad06d.